include/aos/common/tools/fs.hpp

```diff
@@ -421,7 +421,14 @@ class FS {
             return err;
         }
 
-        return text.Resize(buff.Size());
+        err = text.Resize(buff.Size());
```

Comment:
Trim should be done  in caller. ReadFileToString should read what ever is in the file.

Reply:
removed PR